e2e/tests/trustless_relays.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Trustless Relays
+    # Happy test case: An Application requests the account balance of a specific address at a specific height from a Servicer staked for the Ethereum RelayChain, and receives a successful response.
+
+  # ADDPR: Add a servicer staked for the Ethereum relaychain to the genesis file
+  Scenario: Application can send a trustless relay to a relaychain to get an account's balance at a specific height
+    Given the application has a valid ethereum relaychain account
+    Given the application has a valid ethereum relaychain height
+    Given the application has a valid servicer
```

Comment:
Perhaps instead of a `Given`, this should be `And the application has started a session` with a matching servicer?

Pros: It'll add an extra layer of validation
Against: Your test is constrained to something very specific

---

Happy to get a push back if we add a TODO for the other E2E test where we retrieve the session first

Reply:
Thank you for the review, added a TODO item to cover this, as IMO we should add a fetch session command as well to allow an application to get its current session.